### PR TITLE
Feature/inject annotaions

### PR DESCRIPTION
**Pull request by zhhray**
**What this PR does / why we need it**:

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

**Comment by hangyan** [comment]
upgrade 的场景实现了吗

**Comment by zhhray** [comment]
> upgrade 的场景实现了吗

这个提交也可以覆盖到upgrade场景

File: pkg/action/install.go [hunk 1]
```diff
@@ -469,6 +473,77 @@ func (c *Configuration) renderResources(ch *chart.Chart, values chartutil.Values
 	return hs, b, notes, nil
 }
 
+// injectAnnotations will inject annotations to files
+func (c *Configuration) injectAnnotations(originalAnnotations map[string]string, files map[string]string) {
+	if originalAnnotations == nil {
+		return
```
Comment (hangyan) [inline pkg/action/install.go:479]: 为啥会返回?啥情况annotations是nil?是不是大多数chart默认anotations都是nil?

Comment (hangyan) [reply pkg/action/install.go:479]: 哦我知道了，是在captain里生成的。

**Review by hangyan — APPROVED** [review]

**Pull request merged**